spec/queries/project/subscriptions_pledged_value_query_spec.rb

```diff
@@ -8,43 +8,23 @@
 
     subject { described_class.new(project).call }
 
-    context "when the project has registered subscriptions" do
-      context "with paid transactions" do
-        let(:standard_plan) { create(:plan, amount: 3000, projects: [project]) }
-        let(:premium_plan) { create(:plan, amount: 10000, projects: [project]) }
-        let(:subscritpion_for_standard_plan) { create(:subscription, :paid, plan: standard_plan, project: project) }
-        let(:subscritpion_for_premium_plan) { create(:subscription, :canceled, plan: premium_plan, project: project) }
-
-        before do
-          create(:transaction, status: :paid, subscription: subscritpion_for_standard_plan)
-          create(:transaction, status: :paid, subscription: subscritpion_for_premium_plan)
-          create(:transaction, status: :refunded, subscription: subscritpion_for_standard_plan)
-        end
-
-        it "returns the paid transactions formatted amount" do
-          expect(subject).to eq 130
-        end
-      end
+    context "when the project has subscriptions" do
+      let(:subscriptions) { Subscription.all }
 
-      context "without paid transactions" do
-        let(:plan) { create(:plan, projects: [project]) }
-        let(:subscription) { create(:subscription, :paid, plan: plan, project: project) }
-        let!(:pending_payment_transaction) { create(:transaction, :pending_payment, subscription: subscription) }
+      before do
+        plan_30 = create(:plan, amount: 3000)
```

Comment:
What do you think about this:

```Ruby
    before do
        plan_30 = create(:plan, amount: 3000)
        create_list(:subscription, 4, :paid, plan: plan_30, project: project)
    end
```

And remove the `let(:subscription...` on the top of this context.